### PR TITLE
fix!: update go-ipld-prime to master, int64 for x86 support

### PR DESCRIPTION
**Pull request by rvagg**
Fixes: https://github.com/ipld/go-codec-dagpb/pull/3

### CODE REVIEW

**Comment by rvagg** [comment]
I just did `go get github.com/ipld/go-ipld-prime@8fa241e` and `go mod tidy` to get the update. Is that the right way to do a dependency upgrade like this? Should I be preferring to wait for a release to be cut of ipld-prime and using that specific tag?

File: marshal.go [hunk 1]
```diff
@@ -27,7 +27,7 @@ type pbLink struct {
 func Marshal(inNode ipld.Node, out io.Writer) error {
 	// Wrap in a typed node for some basic schema form checking
 	builder := Type.PBNode.NewBuilder()
-	if err := builder.AssignNode(inNode); err != nil {
+	if err := builder.ConvertFrom(inNode); err != nil {
```
Comment (mvdan) [inline marshal.go:30]: fyi, we'll probably undo this rename, but it should be easy for you to undo it here too.

**Review by mvdan — APPROVED** [review]
Yep, what you did sounds fine. You could also do `go get some-module@master` if you don't need a specific commit.

**Comment by rvagg** [comment]
switched to v0.7.0 and undid the `ConvertFrom`

**Pull request merged**